### PR TITLE
[IMP] add button to add device in user form

### PR DESCRIPTION
**Pull request by fmdl**
image

### CODE REVIEW

**Review by yajo — APPROVED** [review]

**Comment by rami-wafaie** [comment]
:+1:

**Review by rami-wafaie — APPROVED** [review]
:+1:
Looks fine for me

**Review by michotm — APPROVED** [review]
tested in runbot, working fine with free-ot app

**Review by houssine78 — APPROVED** [review]
👍 
LGTM
Tested on runbot

**Comment by pedrobaeza** [comment]
Please change version number

**Comment by fmdl** [comment]
@pedrobaeza can you merge ? Tanks

File: auth_totp/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 {
     'name': 'MFA Support',
     'summary': 'Allows users to enable MFA and add optional trusted devices',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```
Comment (elicoidal) [inline auth_totp/__manifest__.py]: not to be picky but I think it should be 10.0.1.1.0 as it requires a module update.

**Comment by pedrobaeza** [comment]
There are now conflict with several files as another PR has been merged. Please solve conflicts (and maybe change correctly the version number) and I'll merge.

**Pull request closed without merging**